webui/react/src/components/Markdown.tsx

```diff
@@ -40,7 +41,13 @@ const MarkdownRender: React.FC<RenderProps> = ({ markdown, placeholder, onClick
   );
 };
 
-const Markdown: React.FC<Props> = ({ editing = false, markdown, onChange, onClick }: Props) => {
+const Markdown: React.FC<Props> = ({
+  disabled = false,
+  editing = false,
+  markdown,
+  onChange,
+  onClick,
+}: Props) => {
   return (
     <div aria-label="markdown-editor" className={css.base}>
       {editing ? (
```

Comment:
question: do we want to remove the ability to open the editor if `disabled === true`? This condition could be `(editing && !disabled) ? ...`